### PR TITLE
add support for watchers

### PR DESCRIPTION
**Pull request by KristofferC**
Fixes https://github.com/JuliaDebug/Debugger.jl/issues/45

### CODE REVIEW

File: src/commands.jl [hunk 1]
```diff
@@ -200,19 +231,23 @@ function execute_command(state::DebuggerState, ::Val{:?}, cmd::AbstractString)
...
+    - `n`: step to the next line\\
+    - `s`: step into the next call\\
+    - `finish`: run to the end of the function\\
+    - `bt`: show a simple backtrace\\
+    - ``` `stuff ```: run `stuff` in the current function's context\\
+    - `fr [v]`: show all variables in the current frame, `v` defaults to 1\\
+    - `w`\\
+        - `w a expr`: add an expression to the watch list\\
+        - `w`: show all watch expressions evaluated in the current function's context\\
+        - `w c [i::Int]`: clear all or the `i`:th watch expression\\
```
Comment (pfitzseb) [inline src/commands.jl]: Might be better if those commands were consistent with e.g. the Pkg REPL mode, so `w add` for adding watchers and `w rm [i]` or `w remove [i]` for removing them.

Comment (KristofferC) [reply src/commands.jl]: Good idea

**Comment by KristofferC** [comment]
Updated

**Comment by KristofferC** [comment]
UI tests fails on nightly. I can't build julia locally and the windows nightlies are offline so I'll deal with this PR later.

**Comment by KristofferC** [comment]
Should be fixed on master. I forgot to extract the parsed expr from `:toplevel`.

**Pull request merged**